Makefile

```diff
@@ -264,7 +264,9 @@ quiet_cmd_ttman = MAN    $@
 
 data		= $(wildcard data/*)
 
-clean		+= *.o ip/*.lo op/*.lo ip/*.so op/*.so cmus libcmus.a cmus.def cmus.base cmus.exp cmus-remote Doc/*.o Doc/ttman Doc/*.1 Doc/*.7 .install.log
+clean		+= *.o *.lo ip/*.lo op/*.lo ip/*.so op/*.so cmus libcmus.a cmus.def \
+		   cmus.base cmus.exp cmus-remote Doc/*.o Doc/ttman Doc/*.1 Doc/*.7 \
+	           .install.log
```

Comment:
ok, but see above.

Reply:
This should not be necessary anymore, `*.lo` has been added in #864.

Reply:
Removed.